.travis.yml

```diff
@@ -39,11 +39,24 @@ matrix:
           env:
             - MATRIX_EVAL="CC=gcc-9 && CXX=g++-9" 
 
+        - os: linux
+          addons:
+              apt:
+                  sources:
+                      - ubuntu-toolchain-r-test
+                  packages:
+                      - g++-9
+                      - libcppunit-dev
+                      - gettext
+                      - libjsoncpp-dev
+          env:
+            - MATRIX_EVAL="CC=gcc-9 && CXX=g++-9"
+
 before_install:
     - eval "${MATRIX_EVAL}"
 
 install:
-    - wget http://www.auto.tuwien.ac.at/~mkoegler/pth/pthsem_2.0.8.tar.gz
+    - wget https://www.auto.tuwien.ac.at/~mkoegler/pth/pthsem_2.0.8.tar.gz --no-check-certificate
```

Comment:
Sorry, I forgot to tell you to disregard the status of the Travis build. It will soon be decommissioned as we are now using Github Actions for CI builds.

Reply:
I just merged #55 that disables the Travis integration. That creates a merge conflict that you can resolve by deleting .travis.yml from your branch when rebasing on/merging with master.
The CI build on Github Actions is now building with and without libcurl4-openssl-dev. We probably also need to build with and without jsoncpp.